atomic_defi_design/qml/Settings/AddCustomCoinModal.qml

```diff
@@ -63,23 +64,53 @@ BasicModal {
         addToConfig(input_contract_address, "contract_address", input_contract_address.field.text)
         addToConfig(input_active,           "active",           input_active.checked)
         addToConfig(input_coingecko_id,   "coingecko_id",   input_coingecko_id.field.text)
+        fields['coinType'] = currentType.coinType
 
         root.config_fields = General.clone(fields)
     }
 
     function reset() {
-        //input_type.currentIndex = 0 // Keep same type, user might wanna add multiple of same type
         input_ticker.field.text = ""
         input_logo.path = ""
         input_name.field.text = ""
         input_contract_address.field.text = ""
         input_active.checked = false
         input_coingecko_id.field.text = "test-coin"
     }
+    property var typeList: ["ERC-20", "QRC-20","BEP-20"]
+    readonly property string general_message: qsTr('Get the contract address from')
+
+    ListModel {
+        id: type_model
+        dynamicRoles: true
+        ListElement {
+            text: "ERC-20"
+            prefix: ""
+            image: "erc"
+            url: "https://etherscan.io/tokens"
+            name: 'Etherscan'
+            coinType: CoinType.ERC20
+        }
+        ListElement {
+            text: "QRC-20"
+            prefix: "0x"
+            image: "qrc"
+            url: "https://explorer.qtum.org/tokens/search"
+            name: 'QTUM Insight'
+            coinType: CoinType.QRC20
+        }
+        ListElement {
+            text: "BEP-20"
+            prefix: ""
+            url: "https://bscscan.com/"
```

Comment:
https://bscscan.com/tokens would save users a click to find the tokens list.

Reply:
solved in https://github.com/KomodoPlatform/atomicDEX-Desktop/pull/907/commits/d9184d7d9d64f0cbe60b22895745df6803c787f5